### PR TITLE
Fix for Http2 reset vulnerability CVE-2023-39325

### PR DESCRIPTION
**Pull request by frzifus**
We can keep our current go version since there is a backport: https://github.com/golang/go/issues/63426

### CODE REVIEW

**Comment by pavolloffay** [comment]
We should bump the operand versions that have the fix

**Comment by pavolloffay** [comment]
> We can keep our current go version since there is a backport: https://github.com/golang/go/issues/63426

The docker image is using `FROM golang:1.21`

**Comment by frzifus** [comment]
> > We can keep our current go version since there is a backport: https://github.com/golang/go/issues/63426
> 
> The docker image is using `FROM golang:1.21`. The link references golang 1.20

There is a lot of 1.20 in use:
```
.github/workflows/changelog.yaml:          go-version: "1.20"
.github/workflows/continuous-integration.yaml:          go-version: "1.20"
.github/workflows/continuous-integration.yaml:          go-version: "1.20"
.github/workflows/e2e.yaml:        go-version: "1.20"
.github/workflows/e2e.yaml:        go-version: "1.20"
.github/workflows/release.yaml:        go-version: "1.20"
.github/workflows/scorecard.yaml:          go-version: "1.20"
```

**Comment by frzifus** [comment]
I assume we have to wait a bit
image

File: bundle/community/manifests/tempo-operator-manager-config_v1_configmap.yaml [hunk 1]
```diff
@@ -23,11 +23,11 @@ data:
     # if you are doing or is intended to do any operation such as perform cleanups
     # after the manager stops then its usage might be unsafe.
     # leaderElectionReleaseOnCancel: true
-    images:
-      tempo: docker.io/grafana/tempo:2.2.1
-      tempoQuery: docker.io/grafana/tempo-query:2.2.1
-      tempoGateway: quay.io/observatorium/api:main-2023-09-13-14e06c6
-      tempoGatewayOpa: quay.io/observatorium/opa-openshift:main-2023-05-24-8e91537
+    images: # fix: CVE-2023-39325 in 87db5b4
+      tempo: docker.io/grafana/tempo:main-87db5b4
```
Comment (pavolloffay) [inline bundle/community/manifests/tempo-operator-manager-config_v1_configmap.yaml:27]:
```
 docker run --rm -it   docker.io/grafana/tempo:main-87db5b4                                                                                                                                                                  125 ↵ ploffay@fedora
Unable to find image 'grafana/tempo:main-87db5b4' locally
docker: Error response from daemon: manifest for grafana/tempo:main-87db5b4 not found: manifest unknown: manifest unknown.
See 'docker run --help'.
```

Comment (frzifus) [reply bundle/community/manifests/tempo-operator-manager-config_v1_configmap.yaml:27]: See https://github.com/grafana/tempo-operator/pull/642#issuecomment-1761691872

**Review by rubenvp8510 — APPROVED** [review]

**Comment by pavolloffay** [comment]
@frzifus any reason why tempo with the fix was not bumped?

**Comment by frzifus** [comment]
It was done here: https://github.com/grafana/tempo-operator/pull/645

**Comment by rubenvp8510** [comment]
Does it has the CVE fix?

**Comment by pavolloffay** [comment]
> It was done here: https://github.com/grafana/tempo-operator/pull/645

https://github.com/grafana/tempo/commits/release-v2.2/ 2.2.3  does not have the fix

**Comment by frzifus** [comment]
uff - let me bump it

**Pull request merged**